framework/OpenMod.Core/Plugins/PluginActivator.cs

```diff
@@ -81,31 +81,49 @@ public IReadOnlyCollection<IOpenModPlugin> ActivatedPlugins
                     throw new ObjectDisposedException(nameof(PluginActivator));
                 }
 
-                var pluginMetadata = assembly.GetCustomAttribute<PluginMetadataAttribute>();
-                if (pluginMetadata == null)
+                var dependencies = assembly.GetReferencedAssemblies();
+                var missingDependencies = dependencies.Where(s => !AppDomain.CurrentDomain.GetAssemblies().Any(c => c.GetName().Name.Equals(s.Name, StringComparison.OrdinalIgnoreCase))).ToList();
```

Comment:
Does it even works with hotloaded assemblies?

Reply:
Looks like it's not:
```ini
[2023-11-05 19:30:03 WRN][OpenMod.Core.Plugins.PluginActivator] Some dependencies for OpenMod.Economy-7a5e45 Version=1.3.12.0 may be missing. Plugin will try to load anyways...
[2023-11-05 19:30:03 WRN][OpenMod.Core.Plugins.PluginActivator] Missing dependencies: OpenMod.Extensions.Economy.Abstractions Version=3.6.0.0, LiteDB Version=5.0.17.0, MySqlConnector Version=2.0.0.0
[2023-11-05 19:30:03 INF][OpenMod.Economy.Economy] [loading] OpenMod.Economy v1.3.12
[2023-11-05 19:30:03 INF][OpenMod.Economy.Economy] Database type set to: 'DataStore'
[2023-11-05 19:30:03 INF][OpenMod.Runtime.OpenModHostedService] > 1 plugins loaded
```

Reply:
yeh working on it